client/components/ReviewRating/RatingBreakDown/ProductBreakDown.jsx

```diff
@@ -17,7 +17,7 @@ const ProductBreakDown = ({ charValue, charName }) => {
     for (let i = 0; i < char.length; i += 1) {
       if (charName.includes(char[i][0])) {
         const index = charName.indexOf(char[i][0]);
-        const position = ((charValue[index] / 5) * 100).toFixed(0).concat('%');
+        const position = ((charValue[index] / 5) * 100 - 5).toFixed(0).concat('%');
```

Comment:
I had to reduce this by 5 because for some reason it was extending past the end of the bar.